linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/account_password_selinux_faillock_dir/bash/shared.sh

```diff
@@ -1,14 +1,15 @@
 #!/bin/bash
 # platform = multi_platform_all
 
-FAILLOCK_DIR=$(grep -oP "^\s*(?:auth.*pam_faillock.so.*)?dir\s*=\s*(\S+)" \
-                    /etc/security/faillock.conf \
-                    /etc/pam.d/system-auth \
-                    /etc/pam.d/password-auth \
-                    | sed -r 's/.*=\s*(\S+)/\1/')
-
-mkdir -p $FAILLOCK_DIR
-
-semanage fcontext -a -t faillog_t "$FAILLOCK_DIR(/.*)?"
-
-restorecon -R -v "$FAILLOCK_DIR"
+FAILLOCK_CONF_FILES="/etc/security/faillock.conf /etc/pam.d/system-auth /etc/pam.d/password-auth"
+
+for dir in $(grep -oP "^\s*(?:auth.*pam_faillock.so.*)?dir\s*=\s*(\S+)" $FAILLOCK_CONF_FILES \
+            | sed -r 's/.*=\s*(\S+)/\1/'); do
+    if ! `semanage fcontext -a -t faillog_t "$dir(/.*)?"`; then
```

Comment:
Why is this command inside backticks?

Reply:
Thanks for asking. Indeed, it is not necessary to use command substitution here as it is.